### ISSUE TITLE
Album art will not upload in newly added music

### ISSUE THREAD

**Issue by Kmarksman** [sll552/DiscordBee#187]
**Describe the bug**
As the title suggests, newly added music will not upload to the custom DiscordBee rich presence.

**To Reproduce**
1. Add new music to MusicBee and observe that it will not display the album cover in rich presence if checked.
(This is for newly added music as of 1/4/2023, music added beforehand will/would display just fine, this is if you add new albums. This of course meant that after troubleshooting by reinstalling MusicBee and/or reconfiguring/readding DiscordBee, none of the album covers would display anymore).

**Expected behavior**
With or w/o uploading album covers, the MusicBee icon should display I'm pretty sure. Neither that nor the album cover will show up.

**Screenshots**
https://prnt.sc/gKCI_n0q2h1h

**Environment (please complete the following information):**
 - MusicBee Version: 3.5.8402 or 3.5.8384
 - Plugin Version: 3.0.4

**Additional context**
Everything else about the plugin works fine, it is just that album covers will not display. I'm not sure if it's a plugin issue or a discord issue.

**Comment by starstriker0** [comment]
im having the same problem, its especially a  problem for people like me whos music taste changes alot and download alot of music

**Comment by Kmarksman** [comment]
> im having the same problem, its especially a problem for people like me whos music taste changes alot and download alot of music

If you go ahead and create a Custom Imgur ID with the details explained on the main post, you should be able to see newly added artwork with a Imgur ID instead of the Discord one. Try that and see if it works, it worked for me.

**Comment by SU042002** [comment]
yes, using imgur id fixed it for me aswell, although it took me a while to figure out how to get that to work. Eventually found a youtube video and managed to make it work
